src/app/(header-layout)/place/components/restaurant/RestaurantMap.tsx

```
@@ -1,12 +1,21 @@
 import { useRef } from 'react'
-import { Map, ZoomControl, MapMarker } from 'react-kakao-maps-sdk'
+import { Map, ZoomControl, MapMarker, CustomOverlayMap } from 'react-kakao-maps-sdk'
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)